src/Addon/ActivationBanner.php

```diff
@@ -17,13 +17,19 @@ class ActivationBanner {
 	 * @return void
 	 */
 	public function show() {
+
+		// Check for Activation banner class.
+		if ( ! class_exists( 'Give_Addon_Activation_Banner' ) ) {
+			include GIVE_PLUGIN_DIR . 'includes/admin/class-addon-activation-banner.php';
+		}
+
 		// Only runs on admin.
 		$args = [
 			'file'              => GIVE_DTD_FILE,
-			'name'              => GIVE_DTD_NAME,
+			'name'              => 'Double the Donation',
```

Comment:
Does this need to be translatable?

Reply:
I say no. It's their brand name. Add-on names I believe could be translated, like Fee Recovery for example. But in this case, that is THEIR brandname and shouldn't be translated.